src/link.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import Feature from '../core/feature.js';
+import ClickObserver from '../engine/view/observer/clickobserver.js';
+import LinkEngine from './linkengine.js';
+import LinkElement from './linkelement.js';
+
+import Model from '../ui/model.js';
+
+import ButtonController from '../ui/button/button.js';
+import ButtonView from '../ui/button/buttonview.js';
+
+import LinkBalloonPanel from './ui/linkballoonpanel.js';
+import LinkBalloonPanelView from './ui/linkballoonpanelview.js';
+
+/**
+ * The link feature.
+ *
+ * It uses the {@link basic-styles.LinkEngine link engine feature}.
+ *
+ * @memberOf link
+ * @extends core.Feature
+ */
+export default class Link extends Feature {
+	/**
+	 * @inheritDoc
+	 */
+	static get requires() {
+		return [ LinkEngine ];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		const viewDocument = this.editor.editing.view;
+
+		// Register document click observer.
+		viewDocument.addObserver( ClickObserver );
```

Comment:
Shorter is better, IMO: `this.editor.editing.view.addObserver( ClickObserver );`